bigchaindb/experimental/core.py

```diff
@@ -0,0 +1,26 @@
+from bigchaindb import App
+from bigchaindb.experimental.utils import ParallelValidator
+from abci.types_pb2 import ResponseDeliverTx
+
+
+CodeTypeOk = 0
+
+
+class ParallelValidationApp(App):
```

Comment:
It is difficult to introduce updates to the ABCI interface this way. For instance, take a look the [chain migration PR](https://github.com/bigchaindb/bigchaindb/pull/2488). Can we abstract away only the validation part somehow?

Reply:
Actually, this may work too, but you need to call `super` in `deliver_tx` as well.

Reply:
Calling `super().deliver_tx(...)` would trigger the old validation again. Regarding the comment on the chain migration PR, we can drop compatibility for now since parallel validation is just an experiment. If there are conflicts or if this code needs to be updated, I can maintain it.